Main2.cpp

```diff
@@ -262,6 +286,7 @@ int main()
             else
             {
                 cout<<"You have lost the war!\n\n";
+                gdp = gdp*0.9;

```

Comment:
gdp has to be gdp/capita. So cities should increase commerce rather than gdp directly

Reply:
Thats what i figured too this morning, ill fix this up and send in a new pull